package org.purpleBean.kmip.codec.ttlv.deserializer.model.core.type;

import org.purpleBean.kmip.codec.ttlv.deserializer.kmip.AbstractKmipDataTypeTtlvDeserializer;
import org.purpleBean.kmip.model.core.type.AttributeValueTextString;

public class AttributeValueTextStringTtlvDeserializer extends AbstractKmipDataTypeTtlvDeserializer<AttributeValueTextString, String> {

    public AttributeValueTextStringTtlvDeserializer() {
        super(AttributeValueTextString.kmipTag, AttributeValueTextString.encodingType, String.class, value -> AttributeValueTextString.builder().value(value).build());
    }
}